src/background.js

```diff
@@ -12,7 +12,7 @@ const timeouts = {};
 
 // Kinto sync and encryption
 
-const client = new KintoClient(KINTO_SERVER);
+const client = new Kinto({remote: KINTO_SERVER, bucket: "default"});
```

Comment:
Should the bucket name be configurable?

Reply:
Also, I think we need to convert from double quotes to single quotes and declare Kinto as a global to appease the ESLint gods:

```sh
/home/travis/build/mozilla/notes/src/background.js
  15:20  error    'Kinto' is not defined        no-undef
  15:57  error    Strings must use singlequote  quotes
```

Reply:
I don't think the bucket should be configurable. The default bucket is "API sugar" to allow each user to get their own bucket while still referring to them in a uniform way. It's the only one which behaves like this.